source/beamline/IDEAS/IDEAS13ElementGeDetector.h

```diff
@@ -56,6 +64,21 @@ public slots:
 
 	/// Vortex detectors do not support clearing
 	virtual bool clear() { return false; }
+
+	void setPeakingTime(double time);
+
+	void setPreampGain(double value);
+
+protected:
+	AMReadOnlyPVControl *ge13ElementRealTimeControl_;
+	AMDetector *ge13ElementRealTime_;
```

Comment:
The real time detector stuff originally was in the beamline class but I moved it out to keep things consistent but I can easily move it back.